tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -116,10 +116,6 @@ describe('DataAdapterMixin', () => {
 
           expect(sessionService.invalidate).to.have.been.calledOnce;
         });
-
-        it('returns true', () => {
-          expect(adapter.handleResponse(401)).to.be.true;
-        });
```

Comment:
Unfortunately the diff is slightly difficult to parse. I pulled this test one `describe` up and it's now in the `when the response status is 401` describe block. Maybe it would be appropriate to pull it up another level (as the same test is in response status is 401 and status is not 401).

I suggest to view the [whole file](https://github.com/jayeff/ember-simple-auth/blob/update_401_error_handling/tests/unit/mixins/data-adapter-mixin-test.js#L103-L148) for better readability of the tests for `handleResponse`.